.github/workflows/build-images-ci.yaml

```diff
@@ -290,7 +290,7 @@ jobs:
 
       # PR updates
       - name: CI Build ${{ matrix.name }}
-        if: ${{ github.event_name == 'pull_request_target' }}
+        if: ${{ github.event_name == 'pull_request_target' || startsWith(github.ref_name, 'ft/') }}
```

Comment:
This one will do nothing because `ft/` is only for `push` events, right?

Reply:
so maybe something like this?
```
        if: ${{ github.event_name == 'pull_request_target' || (github.event_name == 'push' && startsWith(github.ref_name, 'ft/')) }}

Reply:
alright updated the pr. could you take another look 🙏

Reply:
Aaah now I get it.